pkg/registry/registry.go

```diff
@@ -323,10 +340,22 @@ func (c *Controller) maybeCreateForwarder(ctx context.Context, port int) error {
 		return nil
 	}
 
-	_, _ = fmt.Fprintf(c.iostreams.ErrOut, " 🎮 Env DOCKER_HOST set. Assuming remote Docker and forwarding registry to localhost:%d\n", port)
+	_, _ = fmt.Fprintf(c.iostreams.ErrOut,
+		" 🎮 Env DOCKER_HOST set. Assuming remote Docker and forwarding registry to localhost:%d\n", port)
 	return c.socat.ConnectRemoteDockerPort(ctx, port)
 }
 
+func (c *Controller) listContainers(ctx context.Context, filterArgs filters.Args) ([]types.Container, error) {
+	containers, err := c.dockerClient.ContainerList(ctx, types.ContainerListOptions{
+		Filters: filterArgs,
+		All:     true,
+	})
+	if err != nil {
```

Comment:
i'm not sure what this wrapper method is adding?

Reply:
It does more now 😝